src/Changelog/PullRequestChangelogGenerator.php

```diff
@@ -30,8 +31,10 @@ public function getUnreleasedChangelog(): Changelog
         $unreleasedCommits = $this->versionControlSystem->getCommitsSinceLastVersion(self::PULL_REQUEST_PATTERN);
         $unreleasedChanges = array_map([$this, 'createChangeFromCommit'], $unreleasedCommits);
 
+        $filteredChanges = $this->filterChangesLogElements($unreleasedChanges);
```

Comment:
To be very strict this isn't filtering ChangeLog elements yet. Since the changelog is generated on the line below.
Maybe we should name this accordingly; Something like `filterUnreleasedChanges` or even more specific (for understandability) `filterUnreleasedChangesWithMergeTag` or something?

Reply:
Changed it to `filterUnreleasedChanges`